x/ecocredit/basket/features/msg_put.feature

```diff
@@ -95,7 +95,7 @@ Feature: MsgPut
     }
     """
     When the message is validated
-    Then expect the error "invalid batch denom: expected format A00-000-00000000-00000000-000: parse error: invalid request"
+    Then expect the error "invalid batch denom: expected format A00-000-00000000-00000000-000: invalid request"
```

Comment:
Here's an example of what might be the preferred format but the deleted `parse error` might actually make sense to keep as the error in the utility function and the error returned within basic message validation would wrap the returned error with an `invalid request` error.